B.py

```diff
@@ -0,0 +1,3 @@
+
+def hallo():
```

Comment:
bbbbbbbbbbbaaaaaaaaaaaaaakkkkkkkkkkkkkkkaaaaaaaaaaaa
```suggestion
def hello():
```